deploy/chart/catalog_resources/rh-operators/prometheusoperator.0.22.2.clusterserviceversion.yaml

```diff
@@ -158,18 +158,18 @@ spec:
               serviceAccount: prometheus-operator-0-22-2
               containers:
               - name: prometheus-operator
-                image: quay.io/coreos/prometheus-operator@sha256:3daa69a8c6c2f1d35dcf1fe48a7cd8b230e55f5229a1ded438f687debade5bcf
+                image: quay.io/coreos/prometheus-operator:master
```

Comment:
could you remove the changes from this file? this was just for testing out the operatorgroup in that combined branch for the demo, it's not something we want to change just yet